elements/barcode-reader/barcode-reader.js

```diff
@@ -57,36 +61,33 @@ class BarcodeReader extends LitElement {
    */
   render() {
     return html`
-      <div id="hidden">
+      <div class="hidden">
         <div>
           <video
             muted
             autoplay
-            id="video"
+            class="video"
             playsinline="true"
             width="${this.scale}%"
             height="${this.scale}%"
           ></video>
-          <canvas id="canvas" style="display: none; float: bottom;"></canvas>
+          <canvas class="canvas" style="display: none; float: bottom;"></canvas>
```

Comment:
just target the `canvas` element as opposed to class being name of the tag here and in the video tag. There's no other way to deliver this functionality than w/ these specialized HTML primitives so no reason to not use the tag name